brainwave-prediction-app/GUI.py

```diff
@@ -132,19 +134,45 @@ def holding_pattern_window():
 # Define the layout for the Starting Page
 layout1 = [
     [sg.Button('Brainwave Reading', size=(20, 3)),
-     sg.Button('Transfer Data', size=(20, 3), disabled=True),
+     sg.Button('Transfer Data', size=(20, 3), disabled=False),
      sg.Button('Manual Drone Control', size=(20, 3)),
      sg.Button('Holding Pattern', size=(20, 3), disabled=True)]
 ]
 
-# Define the layout for the Transfer Data Page
-layout4 = [[sg.Button(
-    image_filename="/Users/williamdoyle/Documents/GitHub/Avatar/brainwave-prediction-app/images")]]
 
 # Create the windows
 window1 = sg.Window('Start Page', layout1, size=(1200, 800), finalize=True)
-window4 = sg.Window('Transfer Data', layout4, size=(
-    1200, 800), element_justification='c')
+
+
+# Transfer Window Screen
+def transfer_window():
+    # Define the layout of the transfer window
+    transfer_window_layout = [
+        [sg.Text("Insert a file")],
+        [sg.InputText(),
+         sg.FileBrowse("Browser File")],

```

Comment:
Very minor but could change this pair to be on a single line like `[sg.InputText(), sg.FileBrowse("Browser File")],` to be consistent with line 154.